pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[tool.black]
+line-length = 180
```

Comment:
Hmm, 180 seems way too much, is that how many characters some lines have?

Reply:
This was copied from the prior line-length set for flake8. I personally don't like conda or conda-build's line length policies and would much rather we switch to black's standard 88 line length. Would such a change be out of scope here too?

Reply:
I think this might be okay for this PR TBH since line length should only apply to a small number of lines in total.

Reply:
True, do we want to use conda's line length or just use the default from black?

Reply:
Hm, I don't have a strong opinion other than preferring one standard than many. If both conda and conda-build seem unreasonable, we should default to black's default.

Reply:
Ok I went with black's default then, as black suggests we should also consider using [Bugbear's B950](https://github.com/PyCQA/flake8-bugbear#:~:text=tutorial%20for%20details.-,B950,-%3A%20Line%20too%20long) for linting.